jetbrains-core/it/software/aws/toolkits/jetbrains/services/clouddebug/CloudDebugTestCase.kt

```diff
@@ -56,11 +56,15 @@ abstract class CloudDebugTestCase(private val taskDefName: String) {
     @JvmField
     val resourceCache = MockResourceCacheRule()
 
+    @Rule
+    @JvmField
+    val mockRegionProvider = MockRegionProviderRule()
+
     @Before
     open fun setUp() {
         // does not validate that a SSM session is successfully created
         val region = AwsRegion("us-west-2", "US West 2", "aws")
-        MockRegionProvider.getInstance().addRegion(region)
+        mockRegionProvider.addRegion(region)
```

Comment:
since this has to be a legit region, should we use use the `addRegion(Region)` call?

Reply:
The what?

Reply:
`software.aws.toolkits.jetbrains.core.region.MockRegionProviderRule#addRegion(software.amazon.awssdk.regions.Region)`

Reply:
Maybe? I wanted to change as little in here as possible, we can do that later